src/hash.rs

```diff
@@ -242,6 +244,15 @@ impl BlobFormat {
     }
 }
 
+impl From<BlobFormat> for ExportFormat {
+    fn from(value: BlobFormat) -> Self {
+        match value {
+            BlobFormat::Raw => ExportFormat::Blob,
+            BlobFormat::HashSeq => ExportFormat::Collection,
```

Comment:
@rklaehn this is pretty confusing, at least to me, can we simplify/normalize this somehow?

Reply:
Yes, but not in this PR. I will keep it in mind for the blobs v1 api. But in general I don't want collections to be a central part of the store, so some mapping will still have to be done.

Reply:
FWIW, the current version of the example that I pushed (I took over this branch) doesn't depend on this additional change.
So essentially I can remove the `From` impl. I'll push a commit.